sos/plugins/blueprints.py

```diff
@@ -0,0 +1,41 @@
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class Blueprints(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """Composer Blueprints
```

Comment:
It seems a little weird that nothing in here is called "blueprints", apart from the plugin and the sub-command we are running?

Why not "composer", "lorax", or some variant on that? Would we ever want to add any more non-blueprint commands here? Blueprint is a pretty generic name and it would be much easier to justify if the package names or upstream project included it.